src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -1,8 +1,25 @@
+from typing import List, Tuple
+
 from django.db import connection
 
 from sentry.models import GroupOwnerType
 
 
+def format_comment(issues: List[Tuple[str]]):
+    body = "## Suspect Issues\nThis pull request has been deployed and Sentry has observed the following issues:\n\n"
```

Comment:
Could we use string formats?
```
body = """## Suspect Issues
This pull request has been deployed and Sentry has observed the following issues

{issue_list}

<sub>Did you find this useful? React with a 👍 or 👎</sub>"""

single_issue_template = "- **{title}** `{subtitle}` [View Issue]({url})"

def format_comment(issues):
  # ...
  issue_list = "\n".join([single_issue_template.format(title=issue[0], ...)
  return body.format(issue_list=issue_list)
```

Reply:
Also use a multiline string so that it can resemble what the final comment will look like

Reply:
sounds good, thanks!